x/lockup/keeper/lock_test.go

```diff
@@ -45,6 +45,67 @@ func (suite *KeeperTestSuite) TestBeginUnlocking() { // test for all unlockable
 	suite.Require().NotEqual(locks[0].IsUnlocking(), false)
 }
 
+func (suite *KeeperTestSuite) TestBeginForceUnlock() {
+	// coins to lock
+	coins := sdk.Coins{sdk.NewInt64Coin("stake", 10)}
+
+	testCases := []struct {
+		name             string
+		coins            sdk.Coins
+		unlockCoins      sdk.Coins
+		expectSameLockID bool
+	}{
+		{
+			name:             "new lock id is returned if the lock was split",
+			coins:            coins,
+			unlockCoins:      sdk.Coins{sdk.NewInt64Coin("stake", 1)},
```

Comment:
nit: while not a problem here, using the constructor is preferable because it does coin validation
```suggestion
			unlockCoins:      sdk.NewCoins(sdk.NewInt64Coin("stake", 1)),
```